src/Agent.Sdk/Util/ProcessUtil.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Please rename the file name to match the class name

Reply:
I don't think this is a good idea, as more classes may be added in the future.